src/lib/Server/Controller/ContentType.php

```diff
@@ -4,6 +4,8 @@
  * @copyright Copyright (C) Ibexa AS. All rights reserved.
  * @license For full copyright and license information view LICENSE file distributed with this source code.
  */
+declare(strict_types=1);
+
```

Comment:
Due to side effects, we can't add this in 4.6.

Reply:
Can't we after type casting the whole class as I just did?